dart.py

```diff
@@ -3,15 +3,50 @@
 API info at http://api.irishrail.ie/realtime/
 """
 
-import httplib, urllib
+import httplib
+import urllib
 import argparse
+import json
 from xml.etree import ElementTree
 
-
 HOST = "api.irishrail.ie"
 NS = {'irishrail': "http://%s/realtime/" % HOST}
 
 
+class GoogleDistance():
+
+    # Note that this API key is restricted to be used by me (my network) and will (most likely) not work for you.
+    # You have to get your own
+    GDAPI_KEY = "AIzaSyC20WYqcBzBYT8biVPl_pdwbQkED_atuko"
```

Comment:
Could do, but then again, the software would make much more sense hosted somewhere SaaS-style with a UI anyway and then there's no need for an API Key change.